docs/metaed-project/MetaEdOutput/EdFi/Database/PostgreSQL/ODS/Structure/0040-IdColumnUniqueIndexes.sql

```diff
@@ -0,0 +1,192 @@
+CREATE UNIQUE INDEX IF NOT EXISTS UX_a97956_Id ON edfi.AcademicWeek(Id);
```

Comment:
Is there anyway to build the MetaEd output without creating the `edfi` schema and all of its tables?

Reply:
It's all-or-nothing per database.

Reply:
I will look at the filesystem to see if we can gitignore the "edfi" artifacts while keeping the "fizz" artifacts.